microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -123,7 +123,8 @@ Only one installation of the Evergreen WebView2 Runtime is needed for all Evergr
 
 *   For offline clients: _WebView2 Runtime Standalone Installer_ is a full installer that installs the Evergreen WebView2 Runtime in offline environments.
 
-Currently, both the bootstrapper and standalone installer only support per-machine installs, which requires elevation of permissions.  If an installer is run without elevation, the user is prompted to elevate permissions.
+> [!NOTE]
+> The latest bootstrapper and standalone installer support both per-machine and per-user installs. If the installer is run with elevation, the Runtime will be installed as "per-system". Otherwise, the Runtime will be installed as "per-user".  Per-user install is automatically replaced by a per-system install if a per-system Edge Updater is in place.  A per-system Edge Updater is included as part of the Edge browser, except for Canary.
 
```

Comment:
Global: probably change "Edge Updater" to "Microsoft Edge Updater".  Full-text search in repo finds two hits on "Edge Updater":
* "Microsoft Edge pings the Microsoft Edge updater service about the progress of installation and update."
* "The recommended way of receiving WebView2 updates is by using the default Microsoft Edge updater."